docs/usage.md

```diff
@@ -210,6 +217,19 @@ release performed in epoch 1030
 ```
 As with top-down messages, you can get a sense of the time that your message will take to get to the parent by looking at the epoch in which your bottom-up message was triggered (the output of the command), and listing the latest bottom-up checkpoints to see how far it is from being propagated.
 
+The propagation of a bottom-up checkpoint from a child subnet to its parent follows these stages:
+* Validators in the child subnet populate the checkpoint, sign it, and agree on their validity. When validators have agreed on the validity of a checkpoint for a specific epoch, a new `QuorumReached` event is emitted in the child subnet. You can check if a checkpoint for certain epoch has already been signed by a majority of child validators through the following command: `./bin/ipc-cli checkpoint quorum-reached-events --from-epoch 600 --to-epoch 680 --subnet`.
+```console
+# Sample execution
+./bin/ipc-cli checkpoint quorum-reached-events --from-epoch 600 --to-epoch 680 --subnet /r314159/t410ffumhfeppdjixhkxtgagowxkdu77j7xz5aaa52vy
+```
+
+* Once validators have agree on the checkpoint to be submitted in the parent for a specific epoch, relayers need to pick up the checkpoint and submit it in the parent. The following commands can be used to determine what is the state of this submission:
+  * Check if the address of a relayer has already submitted a checkpoint for execution in the parent: `./bin/ipc-cli checkpoint has-submitted-bottomup-height --subnet <SUBNET_ID> --submitted <RELAYER_ADDR>`
```

Comment:
`has-submitted-bottomup-height --subnet <source> --submitted <relayer>` seems odd. Should height be a parameter?

Reply:
@cryptoAtwill, correct me if I am wrong, but this one only checks if a relayer for a specific address has submitted the checkpoint for the latest epoch, right? It doesn't query the submission for a specific epoch (thus the lack of the `epoch` flag). I was also looking for it when I was documenting the command.